src/main/java/net/imagej/ops/geom/geom2d/DefaultEccentricity.java

```diff
@@ -61,10 +62,13 @@ public void initialize() {
 
 	@Override
 	public void compute1(final Polygon input, final DoubleType output) {
-		output.set(majorAxisFunc.compute1(input).getRealDouble() /
-			minorAxisFunc.compute1(input).getRealDouble());
+
+		final double a = majorAxisFunc.compute1(input).get() / 2.0;
```

Comment:
we could operate only on `DoubleType` here: `majorAxisFunc.compute1(input).get().copy().div(2.0)` etc.